httpclient5-testing/src/test/java/org/apache/hc/client5/testing/sync/TestConnectionReuse.java

```diff
@@ -197,8 +204,10 @@ public void testReuseOfAbortedConnections() throws Exception {
             }
         }
 
-        // Expect zero connections in the pool
-        Assertions.assertEquals(0, this.connManager.getTotalStats().getAvailable());
+        // Expect leased connections to be returned
+        Assertions.assertEquals(0, this.connManager.getTotalStats().getLeased());
+        // Expect some connections in the pool
+        Assertions.assertTrue(this.connManager.getTotalStats().getAvailable() > 0);
```

Comment:
Previously this test expected connections to be closed, however the response was fully drained and ready for reuse.